backend/src/models/course.js

```diff
@@ -26,14 +24,4 @@ const schema = new mongoose.Schema({
   ],
 });
 
-function validate(review) {
```

Comment:
No need for Joi validation because courses don't come in anyway from a user input.